src/app/profile/page.tsx

```diff
@@ -40,6 +40,8 @@ export default function Page() {
       await updateProfile(mockProfile.user_id, {
         hours_per_week: 100, // Update the roles field with new data
       });
+
+      console.log('test');
```

Comment:
We probably want to delete all console.log's before merging!

Reply:
ma'am yes ma'am